### ISSUE TITLE
[med] JAWS+Chrome: Many Pages: Multiple elements announced without button role

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5446]
## Action Performed:
1. Using JAWS + Chrome, open the site https://staging.new.expensify.com/
2. Once you're on the landing page, provide an email and activate the button 'continue'.
3. Once the email verification page opens, press Tab key to focus the button 'not given email address?'.
4. Note how the element is announced.

## Expected Result:
Since this is an interactive element, it should be announced with a button role.

## Actual Result:
The element is announced without a button role.

HTML:
`<div dir="auto" class="css-901oao" style="color: rgb(1, 133, 255); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; margin-top: 12px; text-decoration-color: rgb(1, 133, 255);">Not&nbsp;tester.130@gmail.com?</div>`

Other Occurrences:
1. On New Chat modal the entries of recent contacts announced without a button role.
2. On Add payment method flow the "Add payment method" and "Add PayPal account" buttons do not announce their role.
3. On Landing page the "Continue" button is announced with no role.
4. On Profile overlay when trying to save the updated information, the "Save" button is announced with no role.
## Workaround:
No workaround.

## Area issue was found in:
Email verification page, New chat overlay, Add payment method flow, Landing page, Profile overlay

## Failed WCAG checkpoints
4.1.2

## User impact:
The screen reader users are not informed that these are interactive elements and are not aware of their functionality.

## Suggested resolution:
Implement the mentioned elements as native buttons using <button> tag. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

## Platform:
Where is this issue occurring?
- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
![Bug5179534_button_role_not_announced](https://user-images.githubusercontent.com/88733897/134514450-98be1276-aa73-4eda-bead-2dad7c800e4b.jpg)
![Bug5179534_button_role_not_announced_for_recent_contacts](https://user-images.githubusercontent.com/88733897/134514466-ff6cfb3f-3035-4c83-87ad-e670b17513b3.png)

https://user-images.githubusercontent.com/88733897/134514474-b6d96cd0-09a2-4942-a01c-08971fa55609.mp4

https://user-images.githubusercontent.com/88733897/134514486-2b47dbaf-5aec-4339-b0b3-018ca034020a.mp4

https://user-images.githubusercontent.com/88733897/134514546-406f26d6-9d30-409a-8bc3-73b3da1e9184.mp4

https://user-images.githubusercontent.com/88733897/134514560-adcd15f6-09c3-4b94-8852-290bea74d814.mp4

https://user-images.githubusercontent.com/88733897/134514567-078b6055-7298-451b-88f0-2e6ffb62d493.mp4

**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by nazam100** [comment]
Accessibility Issue: The element announced without button role.
Failed WCAG: 1.3.1
Actual Result: The element is announced without it's button role.
Expected Result: The element should announce it's button role.

Attachment:




https://user-images.githubusercontent.com/66008453/139481657-bf66bce9-0933-4dc3-822c-7ecf1a1ff202.mp4

<img width="929" alt="Capture" src="https://user-images.githubusercontent.com/66008453/139481029-c03a7030-715f-4d33-a109-61ce96aa3ad6.PNG">
:
